src/types/num.rs

```diff
@@ -183,11 +173,12 @@ mod int128_conversion {
             impl IntoPy<PyObject> for $rust_type {
                 fn into_py(self, py: Python) -> PyObject {
                     unsafe {
-                        let bytes = self.to_ne_bytes();
+                        // Always use little endian
+                        let bytes = self.to_le_bytes();
```

Comment:
Though it's not related to this PR, I noticed that `to_le_bytes` is added in Rust 1.32. 
Using always little-endian simplified the code a lot.